services/tenant-ui/frontend/src/store/innkeeper/innkeeperOidcStore.ts

```diff
@@ -14,10 +15,10 @@ export const useInnkeeperOidcStore = defineStore('innkeeperOidcStore', () => {
   const _settings: any = {
     authority: config.value.frontend.oidc.authority,
     client_id: config.value.frontend.oidc.client,
-    redirect_uri: `${window.location.origin}/innkeeper`,
+    redirect_uri: API_PATH.OIDC_INNKEEPER_REDIRECT_URI,
     response_type: 'code',
     automaticSilentRenew: false, // don't need to renew for our needs at this point
-    post_logout_redirect_uri: `${window.location.origin}/innkeeper`,
```

Comment:
I'd lean towards leaving window location code out of constants, just for reading the code and knowing exactly what it's doing in the store here... but that's just me, could go either way

Reply:
fair enough. i just started moving everything over but was on the fence about this. I'll move it out since you noticed it too.